src/main/kotlin/dev/supergrecko/vexe/llvm/ir/Module.kt

```diff
@@ -343,29 +342,29 @@ public class Module internal constructor() : Disposable,
      *
      * @see LLVM.LLVMGetNamedMetadataNumOperands
      */
-    public fun getOperandCount(name: String): Int {
+    public fun getNamedMetadataOperandCount(name: String): Int {
```

Comment:
More accurate name for this function as it was not clear that OperandCount related to NamedMetadata